README.md

```diff
@@ -48,38 +49,37 @@ The first endpoint persists a **CrewMember** in the database by calling `crewMem
 ```java
 public String add(CrewMember crewMember) {
     
-    //...
-    //Jakarta Validation
-    //...
+    public String add(CrewMember crewMember) {
```

Comment:
If I'm reading the diff correctly, it will show `public String add(CrewMember crewMember) {` twice